src/__tests__/buildConfig-test.js

```diff
@@ -228,4 +230,143 @@ test("buildConfig extends rules by replacing the prior rule config completely, n
   }).catch(logError)
 })
 
+test("buildConfig correctly handles `ignoreFiles`", t => {
+  const cwd = process.cwd()
+  process.chdir(__dirname)
+
+  buildConfig({
+    ignoreFiles: [
+      "/ignore.css",
+      "ignore-too.css",
+    ],
+    rules: {
+      "at-rule-empty-line-before": [ "always", {
+        expect: ["blockless-group"],
+      } ],
```

Comment:
The repeated lines of rules in these tests are extra noise. Since they aren't being tested, can you reduce them to the bare minimum necessary?